desktop/src/main/java/bisq/desktop/components/PeerInfoIcon.java

```diff
@@ -170,23 +171,27 @@ protected void addMouseListener(int numTrades,
                         Res.get("peerInfo.unknownAge") :
                 null;
 
-        setOnMouseClicked(e -> new PeerInfoWithTagEditor(privateNotificationManager, tradeModel, offer, preferences, useDevPrivilegeKeys)
-                .fullAddress(fullAddress)
-                .numTrades(numTrades)
-                .accountAge(accountAgeFormatted)
-                .signAge(signAgeFormatted)
-                .accountAgeInfo(peersAccountAgeInfo)
-                .signAgeInfo(peersSignAgeInfo)
-                .accountSigningState(accountSigningState)
-                .position(localToScene(new Point2D(0, 0)))
-                .onSave(newTag -> {
-                    preferences.setTagForPeer(fullAddress, newTag);
-                    updatePeerInfoIcon();
-                    if (callback != null) {
-                        callback.avatarTagUpdated();
-                    }
-                })
-                .show());
+        setOnMouseClicked(e -> {
+            if (e.getButton().equals(MouseButton.PRIMARY)) {
```

Comment:
```suggestion
            if (e.isPrimaryButtonDown()) {
```

Reply:
Thank you @xyzmaker123. `isPrimaryButtonDown` doesn't work correctly in this case and Peer Info window won't show up with a simple left click. It will only show up if left button is pressed the exact moment you release an other mouse button on the avatar